.devcontainer/devcontainer.json

```diff
@@ -2,23 +2,21 @@
 // README at: https://github.com/devcontainers/templates/tree/main/src/go
 {
 	"name": "Go",
-	
 	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-	"image": "mcr.microsoft.com/devcontainers/go:1.21-bookworm",
-
+	"image": "mcr.microsoft.com/devcontainers/go:1.20-bookworm",
```

Comment:
Why change this?

Also did you manually update this file or was it automatically updated by any tool?

Reply:
That was a mistake, I meant to update it to 1.22 as well. I've fixed this in the most recent patch